src/postReviewParser.js

```diff
@@ -0,0 +1,21 @@
+import xss from "xss";
+
+export function parser(request) {
```

Comment:
When exporting a single function from a module, I recommend using the same name for the function (here `parser`) as the name of the module (here `postReviewParser`) and exporting it as `default`. That makes things more consistent, especially when importing the function elsewhere.